types/factories/ERC20ReturnTrueMockUpgradeable__factory.ts

```diff
@@ -329,4 +329,4 @@ const _abi = [
671461012e578063395093511461013d575b600080fd5b6100c16101e0565b6040516100ce9190610575565b60405180910390f35b6100f16100e5366004610534565b50506000606655600190565b60405190151581526020016100ce565b6035545b6040519081526020016100ce565b6100f16101213660046104f9565b5050600060665550600190565b604051601281526020016100ce565b6100f161014b366004610534565b610272565b61017061015e36600461055d565b33600090815260656020526040902055565b005b6101056101803660046104a6565b6001600160a01b031660009081526033602052604090205490565b6100c16102b7565b6100f16101b1366004610534565b6102c6565b6101056101c43660046104c7565b506001600160a01b031660009081526065602052604090205490565b6060603680546101ef906105f7565b80601f016020809104026020016040519081016040528092919081815260200182805461021b906105f7565b80156102685780601f1061023d57610100808354040283529160200191610268565b820191906000526020600020905b81548152906001019060200180831161024b57829003601f168201915b5050505050905090565b3360008181526034602090815260408083206001600160a01b038716845290915281205490916102ae9185906102a99086906105c8565b610366565b50600192915050565b6060603780546101ef906105f7565b3360009081526034602090815260408083206001600160a01b03861684529091528120548281101561034d5760405162461bcd60e51b815260206004820152602560248201527f45524332303a2064656372656173656420616c6c6f77616e63652062656c6f77604482015264207a65726f60d81b60648201526084015b60405180910390fd5b61035c33856102a986856105e0565b5060019392505050565b6001600160a01b0383166103c85760405162461bcd60e51b8152602060048201526024808201527f45524332303a20617070726f76652066726f6d20746865207a65726f206164646044820152637265737360e01b6064820152608401610344565b6001600160a01b0382166104295760405162461bcd60e51b815260206004820152602260248201527f45524332303a20617070726f766520746f20746865207a65726f206164647265604482015261737360f01b6064820152608401610344565b6001600160a01b0383811660008181526034602090815260408083209487168084529482529182902085905590518481527f8c5be1e5ebec7d5bd14f71427d1e84f3dd0314c0f7b2291e5b200ac8c7c3b925910160405180910390a3505050565b80356001600160a01b03811681146104a157600080fd5b919050565b6000602082840312156104b7578081fd5b6104c08261048a565b9392505050565b600080604083850312156104d9578081fd5b6104e28361048a565b91506104f06020840161048a565b90509250929050565b60008060006060848603121561050d578081fd5b6105168461048a565b92506105246020850161048a565b9150604084013590509250925092565b60008060408385031215610546578182fd5b61054f8361048a565b946020939093013593505050565b60006020828403121561056e578081fd5b5035919050565b6000602080835283518082850152825b818110156105a157858101830151858201604001528201610585565b818111156105b25783604083870101525b50601f01601f1916929092016040019392505050565b600082198211156105db576105db610632565b500190565b6000828210156105f2576105f2610632565b500390565b600181811c9082168061060b57607f821691505b6020821081141561062c57634e487b7160e01b600052602260045260246000fd5b50919050565b634e487b7160e01b600052601160045260246000fdfea264697066735822122003c131852b9af0c8d5b74c4df027aefda6bd299ebb8831f540fd3b059e3a151b64736f6c63430008040033";
```

Comment:
Add `types` folder to `.gitgnore`

Reply:
Since it has been pushed already, you will have to run `git rm -r --cached types`